sdk/js/src/index.ts

```diff
@@ -148,6 +151,11 @@ export class Client {
     this.websocketBidStatusCallback = bidStatusCallback;
     this.websocketSvmChainUpdateCallback = svmChainUpdateCallback;
     this.websocketRemoveOpportunitiesCallback = removeOpportunitiesCallback;
+    this.websocketCloseCallback =
+      websocketCloseCallback ??
```

Comment:
why not make it optional similar to other callbacks?

Reply:
It's optional but defaults to throwing an error